analysis2_manuscript.tex

```diff
@@ -624,9 +640,10 @@ \section*{Detection of Selective Sweeps}
     than in the CEU sample, which is consistent with the fact that the YRI sample has a larger effective population size and thus both stronger selection and
     more variation to detect a sweep. Further, the population size of YRI has been relatively constant over the last 100,000 years,
     thus the out-of-Africa bottleneck has does not mute the signal of a sweep as much as in the CEU sample. This is consistent with
-    what we know about the effects of demography on the power to detect sweeps \citep[e.g.,][]{simonsen1995properties}.
+    what is known about the effects of demography on the power to detect sweeps \citep[e.g.,][]{simonsen1995properties}.
     Second, the power to detect sweeps is slightly lower when we use a null model with purifying selection acting on exons (consistent with \cite{schrider2020background}) but the effect is not
-    as strong as one might expect. This is likely because the strength of selection acting on exons is relatively weak, and the
+    as strong as one might expect. 
+    \rngcomment{I'm confused by the terminology ``null model'' throughout this section. Are we always testing on simulated data that includes background selection, and ``null model'' refers to how the methods are tuned? Or are we comparing power on data simulated with and without background selection?} This is likely because the strength of selection acting on exons is relatively weak, and the
```

Comment:
the null model here refers to simulations with and without BGS to determine the threshold cutoff for CLR, etc.